internal/params/config.go

```diff
@@ -116,7 +116,7 @@ var (
 		LeaderRotationExternalBeaconLeaders:    EpochTBD,
 		FeeCollectEpoch:                        big.NewInt(1296), // 2023-04-28 07:14:20+00:00
 		ValidatorCodeFixEpoch:                  big.NewInt(1296), // 2023-04-28 07:14:20+00:00
-		HIP30Epoch:                             EpochTBD,
+		HIP30Epoch:                             big.NewInt(2176), // 2023-10-10 10:00:00+00:00
```

Comment:
big.NewInt(2176), // 2023-10-10 10:00:00+00:00 too early ? remember we need to sometime for the pre hip30 epoch for account migration to happen

Reply:
can you update the comment with the correct date and time